index.d.ts

```diff
@@ -849,6 +849,19 @@ declare namespace Eris {
       id: string;
       username: string;
     };
+    team: OAuthTeamInfo | null;
+  }
+  interface OAuthTeamInfo {
+    icon: string | null;
+    id: string;
+    members: OAuthTeamMember;
+    owner_user_id: string;
+  }
+  interface OAuthTeamMember {
+    membership_state: number;
+    permissions: string[];
+    team_id: string;
+    user: { username: string; discriminator: string; id: string; avatar: string | null };
```

Comment:
Can you sort the props alphabetically as well plz

Reply:
This order was copy pasted from elsewhere. Want me to alphabetize there too?

Reply:
Yeah that would be great thanks

Reply:
Broken out into a new `PartialUser` interface with alphabetized keys. Also used in several other places, correcting some other discrepancies as well (in particular, `avatar` is nullable, not optional).